### PR TITLE
Add context encoder

### PR DESCRIPTION
**Pull request by Pramodh-G**

### CODE REVIEW

File: generative_models/ContextEncoder_PyTorch/utils.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from torchvision import transforms
+from torchvision.utils import make_grid
+import torch
+
+
+
```
Comment (nirmal-suthar) [inline generative_models/ContextEncoder_PyTorch/utils.py]: remove stray lines

File: generative_models/ContextEncoder_PyTorch/utils.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  clone_sample[:,:,i:i+mask_size,i:i+mask_size] = gene_image
+  for i in range(clone_sample.size(0)):
+    clone_sample[i] = inv_transform(clone_sample[i])  
+
+  grid_image = make_grid(clone_sample,nrow=4)
+  grid_samples = make_grid(samples,nrow=4)
+  writer.add_image('original images',grid_samples,current_step)
+  writer.add_image('test images',grid_image,current_step)
+
+  return
```
Comment (nirmal-suthar) [inline generative_models/ContextEncoder_PyTorch/utils.py]: no need to use return statement

File: generative_models/ContextEncoder_PyTorch/main.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+        #train generator
+        optimizer_G.zero_grad()
+
+        gen_parts = generator(masked_imgs)
+
+        #calculate loss
+        g_adv = adversarial_loss(discriminator(gen_parts), valid)
+        g_pixel = pixelwise_loss(gen_parts, masked_parts)      
+        g_loss = 0.001 * g_adv + 0.999 * g_pixel
```
Comment (nirmal-suthar) [inline generative_models/ContextEncoder_PyTorch/main.py]:
```suggestion
        g_loss = lambda_adv * g_adv + (1 - lambda_adv) * g_pixel
```
and add lambda_adv to parser arg

**Review by nirmal-suthar — APPROVED** [review]

**Comment by Pramodh-G** [comment]
made changes,please review.

**Pull request merged**